lib/httparty/request.rb

```diff
@@ -12,7 +12,7 @@ class Request #:nodoc:
       Net::HTTP::Copy
     ]
 
-    SupportedURISchemes  = [URI::HTTP, URI::HTTPS, URI::Generic]
+    SupportedURISchemes  = ['http', 'https', nil]
```

Comment:
I'm starting to have second thoughts on this and I've detected a false PASS Webcal URI test. In the end, all requests are done through HTTPS o HTTP no matter which scheme is originally specify.

Wouldn't it make sense to remove this supported URI schemes list? It all started with this [pull request](https://github.com/jnunemaker/httparty/pull/164).